packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -30,14 +32,52 @@ $usa-banner-font-family: $font-sans;
   }
 }
 
+.ds-c-usa-banner__header--mobile {
+  padding: 0;
+
+  .ds-c-usa-banner__button {
+    display: flex;
+    font-family: $usa-banner-font-family;
+    padding: $spacer-1 $spacer-2 $spacer-1 $spacer-2;
+    width: 100%;
+  }
+
+  &.ds-c-usa-banner__header--expanded {
+    .ds-c-usa-banner__button {
+      padding-bottom: 0;
+    }
+  }
+
+  .ds-c-usa-banner__cta-wrapper {
+    display: block;
+  }
+}
+
+.ds-c-usa-banner__collapse-banner-container {
+  align-items: center;
+  background-color: $usa-banner-close-image-background-color;
+  display: flex;
+  height: $usa-banner-mobile-close-size;
+  justify-content: center;
+  margin-left: auto;
+  margin-right: -$spacer-2;
+  margin-top: -$spacer-1;
+  width: $usa-banner-mobile-close-size;
+
+  .ds-c-icon {
+    height: $font-size-lg;
+    width: $font-size-lg;
+  }
+}
+
 .ds-c-usa-banner__header-flag {
   height: 11px;
   margin-right: $spacer-1;
   width: $spacer-2;
 }
 
 .ds-c-usa-banner__header-text {
-  display: table;
+  display: inline;
```

Comment:
It's curious this layout worked before using `table` - I was under the impression `table` had a block-like behavior instead of inline.